decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/presentation/error/ExceptionHandlers.java

```diff
@@ -21,16 +21,21 @@
 import org.springframework.web.server.ResponseStatusException;
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.web.servlet.error.ErrorController;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
 import org.springframework.stereotype.Controller;
 
 @Controller
 @ControllerAdvice
 public class ExceptionHandlers implements ErrorController {
 
+    @Value("${message.empty}")
+    private String EMPTY_MSG;

```

Comment:
Why do we need an empty message and why does it have to be loaded from properties if it's empty?

Reply:
Or is this maybe if a ISO 20022 is empty or something like that and the "EMPTY_MSG" is the custom message if the ISO 20022 message is empty?

Reply:
That is it. If the HTTP message body is 0 bytes, Spring throws an exception and a 500 status will be returned. I thought it would be more appropriate to return a 400 status instead.